File: Drink.py

class Drink:

    def __init__(self, name, value):
        self.__name = name
        self.__value = value

    def getName(self):
        return self.__name

    def getValue(self):
        return self.__value

    def __str__(self):
        return f"Напиток: {self.__name}  Объём {self.__value}"

File: Drink.java
package lesson_4;


public class Drink {
    private String name;
    private float value;

    public Drink(String name, float value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public float getVolume() {
        return value;
    }

    public void setVolume(int value) {
        this.value = value;
    }

    public String toString() {
        return String.format("%s  %.1f", name, value);
    }
}